lib/ui/homepage.dart

```diff
@@ -1,11 +1,10 @@
-import 'package:flupresso/model/Profile.dart';
-import 'package:flupresso/model/services/state/CoffeeService.dart';
-import 'package:flupresso/model/services/state/ProfileService.dart';
+import 'package:flupresso/model/services/state/coffee_service.dart';
+import 'package:flupresso/model/services/state/profile_service.dart';
 import 'package:flupresso/service_locator.dart';
-import 'package:flupresso/ui/screens/CoffeeScreen.dart';
-import 'package:flupresso/ui/screens/MiscScreen.dart';
+import 'package:flupresso/ui/screens/coffee_screen.dart';
+import 'package:flupresso/ui/screens/water_screen.dart';
 import 'package:flutter/material.dart';
-import 'Theme.dart' as Theme;
+import 'theme.dart' as theme;

```

Comment:
It seems like the common practice is still "Theme" in uppercase

Reply:
Ya, this felt weird doing it, meant to go look more into this. It looks like maybe the best practice is to get the theme from the context which avoids this import niggle?

https://flutter.dev/docs/cookbook/design/themes